src/PointwiseFunctions/AnalyticSolutions/GhGrMhd/InstantiateWrappedGr.cpp

```diff
@@ -0,0 +1,17 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrappedGr.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrappedGr.tpp"
+#include "PointwiseFunctions/AnalyticSolutions/GrMhd/AlfvenWave.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GrMhd/KomissarovShock.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GrMhd/SmoothFlow.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+
+GENERATE_INSTANTIATIONS(WRAPPED_GR_INSTANTIATE,
+                        (grmhd::Solutions::AlfvenWave,
```

Comment:
Do you think it would be worth adding some generic test function that checks say that the `WrappedGr` solution produces the expected GRMHD variables? That is,
```cpp
GrmhdSoln soln{};
WrappedSoln w_soln{};
for_each<hydro::grmhd_tags>(
CHECK(get<tag>(soln) == get<tag>(w_soln));
);
```
I'm a bit hesitant to suggest checking the GH variables like this because that's literally just a re-implementation of the source tree, and presumably that is actually verified by the test for the `WrappedGr` solution.